libsplinter/src/admin/service/mod.rs

```diff
@@ -221,7 +221,9 @@ impl AdminService {
     /// circuits should be initialized using the service orchestrator. This may not include all
     /// services if they are not supported locally. It is expected that some services will be
     /// started externally.
-    fn restart_services(&self) -> Result<(), ServiceStartError> {
+    ///
+    /// Also creates the requests the connection for members of the circuits and proposals
```

Comment:
This is a strange sentence.  I'm not sure what the intent is.

Reply:
whoops, that was a combo of two diff sentences. fixed